dice_scraper.rb

```diff
@@ -0,0 +1,66 @@
+require 'rubygems'
+require 'bundler/setup'
+require 'nokogiri'
+require 'mechanize'
+require 'csv'
+require_relative 'job'
+
+class DiceScraper
+
+  attr_accessor :agent
+  attr_reader :page, :job_elements, :search_form
+
+  def initialize
+    @agent = Mechanize.new
+    @page = agent.get('http://www.dice.com')
+    @search_form = page.form_with(:action=> "/jobs")
+    set_query "Ruby on Rails"
+    set_location "94612"
+    results = agent.submit(search_form)
+    @job_elements = results.search("div[@class='serp-result-content']")
```

Comment:
You probably remember this from yesterday, but this is the same as div.serp-result-content